src/core/tests/frontend/onnx/telemetry.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "telemetry.hpp"
+
+#include "onnx_utils.hpp"
+
+using namespace ngraph;
```

Comment:
Looks like we don't need to have ngraph namespace here

Reply:
fixed